### DISCUSSION TITLE
Confirm Wanted: Is This An Official Website?

### DISCUSSION THREAD

**Discussion by RainoutOvO** [beyondcode/herd-community#948] [Q&A] [answered]
https://herd.laravel.net.cn/
This website looks very similar to herd.laravel.com, but is localized in CN.
Please let me know if this is an official website so that I can do the shopping worryless there.
Many thanks.

**Comment by mpociot** [comment, marked as answer]
Thank you for reporting this site with us. It is not an official Chinese site that we provide

**Reply by RainoutOvO** [reply]
Thank you very much for your response! It has been extremely helpful to me!
